src/lektorium/repo/memory.py

```diff
@@ -22,6 +22,16 @@ def edit_url(self) -> Optional[str]:
         return self.get('edit_url', None)
 
 
+VALID_MERGE_REQUEST = {
```

Comment:
It looks like test data. Why is it here and not in tests?

Reply:
It is like a stub for developing without real gitlab server

Reply:
all the data around used for UI testing and it is special mode of running application instance. we are aware about that, but this will be fixed later, not in this change.